src/Components/Header/Popup/Popup.js

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+import CloseIcon from '@mui/icons-material/Close';
+import './PopupStyle.css';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+function Popup({ show }) {
+
+    const navigate = useNavigate();
+
+    const [user, setUser] = useState({ email: '', password: '' })
+    const [error, setError] = useState('');
+
+    const handleChange = event => {
+        setUser({ ...user, [event.target.name]: event.target.value });
+    }
+    const handleLogin = event => {
+        event.preventDefault();
+        console.log(user);
+        sendUserInfo();
+    }
+
+    const sendUserInfo = async () => {
```

Comment:
You are trying to call the function `sendUserInfo` before it is defined. Although react doesn't through any error but this is bad practice.

`sendUserInfo` doesn't give a message about what this function is doing. what you can do instead is change 

handleLogin -> handleSubmit
sendUserInfo -> handleLogin 

or anything that suits

Reply:
Corrected both things as suggested